configs/mainnet/hubble.json

```
@@ -1032,5 +1032,11 @@
     "oracle_mapping": "ELrhqYY3WjLRnLwWt3u7sMykNc87EScEAsyCyrDDSAXv",
     "oracle_type": "PythEMA",
     "max_age": 200
+  },
+  "198": {
+    "label": "TWAP Switchboard HBB/USD",
```

```suggestion
    "label": "TWAP HBB/USD",
```

I find confusing to have switchboard in the name while it is a scope twap.